variables.tf

```diff
@@ -9,14 +9,51 @@ variable "subnet_type_tag_value_format" {
   type        = "string"
 }
 
-variable "region" {
-  type        = "string"
-  description = "AWS Region (e.g. `us-east-1`)"
+variable "max_subnet_count" {
+  default     = "-1"
+  description = "The maximum number of subnets to deploy. 0 for none, -1 to match the number of az's in the region, or a specific number"
 }
 
-variable "max_subnet_count" {
-  default     = 0
-  description = "Sets the maximum amount of subnets to deploy.  0 will deploy a subnet for every provided availablility zone (in `availability_zones` variable) within the region"
+locals {
```

Comment:
remove `locals` from `variables` and place into other TF files, e.g `main.tf`

Reply:
I have revised.